pkg/agent/controller/networkpolicy/reconciler.go

```diff
@@ -446,13 +515,37 @@ func (r *reconciler) computeOFRulesForAdd(rule *CompletedRule, ofPriority *uint1
 	r.lastRealizeds.Store(rule.ID, lastRealized)
 
 	ofRuleByServicesMap := map[servicesKey]*types.PolicyRule{}
-
+	groupAddresses := r.getIGMPGroupAddress(rule)
+	isIGMP := len(groupAddresses) > 0
```

Comment:
why it's IGMP when groupAddresses are not empty, doesn't it have `service.Protocol == IGMP` to decide?

Reply:
change to check `service.Protocol == IGMP`